components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dto/OAuth2AccessTokenReqDTO.java

```diff
@@ -52,6 +53,7 @@ public class OAuth2AccessTokenReqDTO {
     private HttpServletRequestWrapper httpServletRequestWrapper;
     private List<String> authenticationMethodReferences = new ArrayList<>();
     private OAuthClientAuthnContext oAuthClientAuthnContext;
+    private Map<String, String> customParameters;
```

Comment:
don't need to define as custom.
```suggestion
    private Map<String, String> parameters;
```

Reply:
changed